pkg/apis/messaging/v1beta1/kafka_channel_lifecycle_test.go

```diff
@@ -373,6 +526,169 @@ func TestChannelIsReady(t *testing.T) {
 	}
 }
 
+func TestDistributedChannelIsReady(t *testing.T) {
```

Comment:
There are a lot of tests here, but not any for the "Unknown" statuses (MarkDispatcherDeploymentUnknown, etc). Are those pesky to test? Not a big loss if it's complicated but if it's just a matter of having a "deploymentStatusUnknown" as well it might be good to add them.

Reply:
Yeah, I noticed that - carryover from the legacy test.  Not sure the check buys much but I will enhance it to do so - helps the GA coverage numbers if nothing else : )